sdk/keyvault/platform-matrix.json

```diff
@@ -12,8 +12,8 @@
     {
       "Agent": {
         "ubuntu-20.04": {
-          "OSVmImage": "MMSUbuntu20.04",
-          "Pool": "azsdk-pool-mms-ubuntu-2004-general"
+          "OSVmImage": "env:LINUXVMIMAGE",
```

Comment:
Is this valid syntax? If so, 👍.

Reply:
@heaths yes it's a custom syntax for the matrix config that we added for this scenario: https://github.com/Azure/azure-sdk-tools/blob/main/doc/common/matrix_generator.md#environment-variable-references